src/VCSWebRenderer.ts

```diff
@@ -308,11 +308,11 @@ export default class DailyVCSWebRenderer {
     };
   }
 
-  private placeVideoSourceInDOM(el: HTMLElement, trackId: string) {
+  private placeVideoSourceInDOM(el: HTMLElement, participantId: string) {
```

Comment:
How does this distinguish between camera and screen (and perhaps custom) tracks? Or is this (multiple video tracks per participant) not a thing a VCS?
In case we need to distinguish, we might want to add another parameter `trackType` and glue that into the id. Then it might be more meaningful to name the attribute `data-video-id` only.

Reply:
@Regaddi this id appends `_sshare` to screenshares with this function https://github.com/daily-co/daily-vcs-web/blob/2662a0a90793e13d3bb7804f30b678882390b2a4/src/lib/videoUtils.ts#L16-L17 👍

Reply:
Ah great! 👍